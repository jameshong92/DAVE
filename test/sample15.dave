// test record defintion
int main() {
    rec test = new rec(age: 22, score: 3.88, name: "unknown");
    print(test);
    print(test.length);
}