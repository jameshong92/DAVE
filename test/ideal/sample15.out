age	score	name	
22	3.88	unknown	

3
